README.md

```diff
@@ -3,14 +3,26 @@
 # Hearth
 HEARTH (noun): the floor of a '[FHIR](http://hl7.org/fhir/)'place. A fast FHIR-compliant server focused on longitudinal data stores.
 
-This project aims to provide a fast and lightweight FHIR server that also supports some of the FHIR-based IHE profiles. It is still in the early stages of development, follow the project to stay informed. Any contributions are welcomed!
+This project aims to provide a fast and lightweight FHIR server that also supports some of the FHIR-based IHE profiles. It is still in the early stages of development, follow the project to stay informed.
 
-Our high level plan for the project can be found [here](https://docs.google.com/document/d/1wJr-A0xJFEwwR9y5c5tVGb0_rH7IQFBJRhMNRU31Fis/edit?usp=sharing).
+We do our best to update this project when we have projects with funding that are using it. Any contributions are welcomed and encouraged! Help us make this something great.
 
 # Documentation
 For more information regarding the capabilities of Hearth and how to get working with it please refer to the [wiki documentation](https://github.com/jembi/hearth/wiki)
 
 # Usage
+
+## Using docker compose
+
+**Note:** Requires docker and docker-compose to be installed
```

Comment:
Maybe we can link to the docker docs for the installation process to make it easier for someone that isnt familiar with docker
https://docs.docker.com/install/
https://docs.docker.com/compose/install/